### DISCUSSION TITLE
Window Capture in WinUI

### DISCUSSION THREAD

**Discussion by karmelcorn** [visioforge/.Net-SDK-s-samples#19] [General]
I was able to compile the `VisoForge.VideoCapture.Demos.CS WinUI.sln` however the live-preview does not show up. Was anyone able to compile and run the application successfully?

**Comment by visioforge** [comment]
Do you have something in the log?

![2024-02-20_11-05-56](https://github.com/visioforge/.Net-SDK-s-samples/assets/4507925/a5971126-7128-44f7-8960-97188a0f7244)

**Comment by visioforge** [comment]
Please check the updated sample. We have found an issue - the best format was used by default, and the combo box was ignored. The sample code issue, not SDK.
